Jint/Native/JsSet.cs

```diff
@@ -73,7 +73,7 @@ internal void ForEach(ICallable callable, JsValue thisArg)
         _engine._jsValueArrayPool.ReturnArray(args);
     }
 
-    internal ObjectInstance Entries() => _engine.Realm.Intrinsics.SetIteratorPrototype.ConstructEntryIterator(this);
+    public ObjectInstance Entries() => _engine.Realm.Intrinsics.SetIteratorPrototype.ConstructEntryIterator(this);
```

Comment:
Are `Entries` and `Values` really needed, they don't expose meaningful API outside of Jint's logic at the moment?

Reply:
No, right enough. They're not really needed elsewhere.

Reply:
Changed back to internal.